DataFormats/CTPPSReco/interface/CTPPSLocalTrackLite.h

```diff
@@ -11,75 +11,147 @@
 
 #include <cstdint>
 
+#include "DataFormats/CTPPSReco/interface/CTPPSPixelLocalTrack.h"
+
+
 /**
  *\brief Local (=single RP) track with essential information only.
  **/
 class CTPPSLocalTrackLite
 {
   public:
-    CTPPSLocalTrackLite() : rpId(0), x(0.), x_unc(-1.), y(0.), y_unc(-1.), time(0.), time_unc(-1.)
+
+    CTPPSLocalTrackLite() : rpId(0), x(0.), x_unc(-1.), y(0.), y_unc(-1.), tx(999.), tx_unc(-1.), ty(999.), ty_unc(-1.), chiSquaredOverNDF(-1.), reco_info(CTPPSPixelLocalTrack::invalid), numberOfPointsUsedForFit(0), time(0.), time_unc(-1.)
```

Comment:
please add some newlines for readability

Reply:
Did it, also in another few places where lines were too long